tools/bismark/bismark_pretty_report/bismark2report_wrapper.xml

```diff
@@ -1,25 +1,16 @@
-<tool id="bismark_pretty_report" name="Bismark Pretty Report" version="0.16.3">
+<tool id="bismark_pretty_report" name="Bismark Pretty Report" version="0.19.1" profile="17.01">
```

Comment:
18.01?

Reply:
this particular one doesn't need 18.01 (no bam in/out) but 17.01 for the updated stdio error checks